packages/components/src/custom-select-control/index.js

```diff
@@ -141,7 +141,14 @@ export default function CustomSelectControl( {
 					className="components-custom-select-control__button-icon"
 				/>
 			</Button>
-			<ul { ...menuProps }>
+			{ /* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */ }
+			<ul
+				{ ...menuProps }
+				onKeyDown={ ( e ) => {
```

Comment:
Could we refactor this inline function into a `handleKeyDown` function? We could even use the `useCallback` hook to memoize (with `menuProps` as the only dependency)

Reply:
Good suggestion, thanks!